### ISSUE TITLE
templates

### ISSUE THREAD

**Issue by jyeatman** [yeatmanlab/pyAFQ#170]
For researchers working with infant data we should make it easy to switch out the MNI template for pediatric templates. If a different template is selected, then the ROIs and probability maps need to be transformed from MNI -> custom template

**Comment by arokem** [comment]
This is already implemented through the `reg_template` key-word argument (e.g., [here]( https://github.com/yeatmanlab/pyAFQ/blob/master/AFQ/segmentation.py#L176)), but it would be good to test this with some real data. Do you have a pointer to a pediatric template we could use to try this out?

**Comment by jyeatman** [comment]
One other step we might need to include. If someone supplies another template then we need to warp the template ROIs from the MNI template to the supplied template

**Comment by arokem** [comment]
Oh yes - you're right. I missed that. 

Let me see that I understand what would need to happen: 

- Register custom template to MNI template (affine? non-linear?) 
- Move waypoint ROIs and probability maps to the custom template. 
- Use custom template and moved waypoint ROIs, instead of default.

Would that cover everything?

**Comment by jyeatman** [comment]
yes exactly. Probably doing this with affine + sym. Affine would probably be fine. But I also don't see a reason not to use the "optimal" registration that you already implemented. Though one issue is that the contrast in baby brains is a bit crazy. So in that sense affine might actually be ideal
